src/System.ServiceModel.Tests.Common/src/BaseAddress.cs

```diff
@@ -5,6 +5,29 @@
 
 public static class BaseAddress
 {
+#if USE_FIDDLER
+    // Base address for testing nonexistent endpoints
```

Comment:
Please update the Bridge bug to request configurability of base address to allow things like this.  Also please update #29 to remove this code when we do it right.